README.md

```diff
@@ -217,4 +217,6 @@ $font-family-secondary: 'PT Sans', sans-serif;
 - While our app is designed for desktop use and not mobile use, we could make its design more responsive (it already is responsive, but it is not as mobile friendly as the user might want)
 - While our app is designed for personal use and not social networking, we could build out the profile section to let users upload a profile image and about text
 - Include more tests and a seeder file for development purposes
-- Create a separate organization page on GitHub to host the repositories, instead of having them hosted by a specific team member
\ No newline at end of file
+- Create a separate organization page on GitHub to host the repositories, instead of having them hosted by a specific team member
+
+**READY FOR CODE REVIEW**
```

Comment:
I am reviewing!

Reply:
Please use the [updated-code-review branch](https://github.com/jtreeves/budget-frontend/tree/updated-code-review) to see reviews made over the weekend.